### DISCUSSION TITLE
About linking other CAE software

### DISCUSSION THREAD

**Discussion by xiberJ** [simnibs/simnibs#167] [2. General]
hi ，developers

Can we import the model after simnibs has defined the electrode position into other CAE software, or combine other types of software to solve it?

thanks,
jiangyu

**Comment by jduemose** [comment]
Hi Jiangyu

First off, I converted your issue to a discussion instead. As to your question, I would assume that as long as the software you intend to use is able to read the geometry file (or can be converted to one that it can read) and that our geometry doesn't violate any assumptions/checks that the software might impose, then I don't see why you shouldn't be able to solve your problem in another software. Compartments are labeled separately (volumes and surfaces) so I assume you should have the information you need (e.g., to apply boundary conditions).

Best,
Jesper

**Reply by xiberJ** [reply]
hi，Jesper
I would like to know whether it is possible to import the model into a general CAE software, add some impedance data to the brain tissue after segmentation, and then simulate and obtain some electric field results under different stimulation frequencies？Thanks.

**Reply by jduemose** [reply]
You might have to adapt the mesh somehow (perhaps just the format in which it is stored, perhaps some properties) but I would image this will depend on what CAE you are going to use. I am afraid I can't be more specific than that. I would probably just try and see if (and if so what) your CAE complains about as a first step.

**Reply by xiberJ** [reply]
Thank you very much for your reply. May I ask whether the header model file generated by Charm tool can be in step, stp and other similar formats?

**Reply by TorgeW** [reply]
No, its only in the msh format. We don't provide any conversion from the formats you mentioned.
Please node that SimNIBS requires tetrahedral meshes and the geometries stored in the formats mentioned by you are usually more abstract. So the first task for you would be to ensure that you model only contains tetrahedra and triangles, after that one could think about how to convert that.
